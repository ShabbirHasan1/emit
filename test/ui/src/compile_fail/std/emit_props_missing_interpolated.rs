fn main() {
    emit::emit!("template {x}");
}
