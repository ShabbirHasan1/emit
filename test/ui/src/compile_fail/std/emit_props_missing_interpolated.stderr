error[E0425]: cannot find value `x` in this scope
 --> src/compile_fail/std/emit_props_missing_interpolated.rs:2:28
  |
2 |     emit::emit!("template {x}");
  |                            ^ not found in this scope
